x/ibc-rate-limit/.cargo/config.toml

```diff
@@ -0,0 +1,3 @@
+[alias]
```

Comment:
why is there a cargo file? also i can see that both `x/ibc-rate-limit/bytecode` and `x/ibc-rate-limit/artifacts` contain `rate_limiter.wasm`, is that intentional?

Reply:
The original contract wasn't optimised so i optimised it myself leaving 2 versions in the module until further discussions. i'll leave only optimised version for now